static/js/reducers/coupons.js

```diff
@@ -43,6 +46,8 @@ export const coupons = (state: CouponsState = INITIAL_COUPONS_STATE, action: Act
     };
   case RECEIVE_FETCH_COUPONS_FAILURE:
     return { ...state, fetchGetStatus: FETCH_FAILURE};
+  case SET_ACTIVE_COUPON:
```

Comment:
In any case this race condition will be triggered by my future PR so I'll take care of this edge case